src/languages/es.js

```diff
@@ -167,6 +167,7 @@ export default {
         sizeNotMet: 'El archivo adjunto debe ser mas grande que 240 bytes.',
         wrongFileType: 'El tipo del archivo adjunto es incorrecto',
         notAllowedExtension: 'Este tipo de archivo no está permitido',
+        folderNotAllowedMessage: 'No se permite subir carpetas. Pruebe con un archivo diferente.',
```

Comment:
This translation seems off to me. Did you request it to be translated, e.g. https://expensify.slack.com/archives/C01GTK53T8Q/p1641913188292300

Reply:
Requested Now.

Reply:
Got translation from @rocio https://expensify.slack.com/archives/C01GTK53T8Q/p1690482354912629